RetrospectiveExtension.Frontend/vss-extension-dev.json

```diff
@@ -72,10 +72,6 @@
       "path": "src",
       "addressable": true
     },
-    {
```

Comment:
curious why the `azure-devops-extension` library was not included here as well

Reply:
I'm not sure about the old vss sdk and why it's there from the beginning. However, for the new one, there are 2 reasons:

- When we use webpack to create the bundle file, it's already contained all info it needs from those sdks/apis.
- The sdk itself is just a shim to invoke the underlying services which are only available in an Azure DevOps environment so I guess that there is no need for them in runtime. Also, the sample codes available don't include them.